app/Image/Files/BaseMediaFile.php

```diff
@@ -326,4 +344,32 @@ public static function assertIsSupportedOrAcceptedFileExtension(string $extensio
 			throw new MediaFileUnsupportedException(MediaFileUnsupportedException::DEFAULT_MESSAGE . ' (bad extension: ' . $extension . ')');
 		}
 	}
+
+	/**
```

Comment:
If those two functions are only going to be used in Downloaded files, it makes more sense to place them there, idem for the `MIME_TYPES_TO_FILE_EXTENSIONS`